### DISCUSSION TITLE
Support github markdown alert format

### DISCUSSION THREAD

**Discussion by zookini** [observablehq/framework#1215] [Ideas]
Github markdown [alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) and Observable markdown [notes](https://observablehq.com/framework/markdown#notes) are pretty much the same thing but with a different syntax. Can Observable add support for the github syntax?

**Comment by mbostock** [comment]
You can do this using the [**markdownIt** config option](https://observablehq.com/framework/config#markdownit) to register a Markdown plugin. Here are the steps:

1. Install [markdown-it-github-alerts](https://github.com/antfu/markdown-it-github-alerts), say with `npm install`.

2. Import the plugin in your `observablehq.config.js`, and assign it to the `markdownIt` option. Also declare a custom stylesheet `style.css` that you will use to add the corresponding CSS classes.

```js
import MarkdownItGitHubAlerts from "markdown-it-github-alerts";

export default {
  style: "style.css",
  markdownIt: (md) => md.use(MarkdownItGitHubAlerts)
};
```

3. In the custom stylesheet `style.css`, import the corresponding stylesheets from markdown-it-github-alerts, as well as any other styles you want, including Framework’s default styles and the desired theme.

```css
@import url("observablehq:default.css");
@import url("observablehq:theme-air.css") (prefers-color-scheme: light);
@import url("observablehq:theme-near-midnight.css") (prefers-color-scheme: dark);
@import url("markdown-it-github-alerts/styles/github-colors-light.css");
@import url("markdown-it-github-alerts/styles/github-colors-dark-media.css");
@import url("markdown-it-github-alerts/styles/github-base.css");
```

You can test with page like so:

```md
> [!NOTE]
> Highlights information that users should take into account, even when skimming.
```

Which should look like this:
<img width="1110" alt="Screenshot 2024-04-18 at 3 09 40 PM" src="https://github.com/observablehq/framework/assets/230541/589a74b7-db27-4e96-8dcb-a9f19458c2e5">

**Reply by zookini** [reply]
Thanks Mike that looks good
